starky/src/get_challenges.rs

```diff
@@ -79,27 +71,21 @@ where
 {
     // TODO: Should be used later in compression?
     #![allow(dead_code)]
```

Comment:
Is this still dead code?

Reply:
Still unused for now (see `TODO: Deal with the compressed stuff.`)

Reply:
Perhaps file a ticket and add the ref here so we don't forget? Up to you.